pandas/tests/test_generic.py

```diff
@@ -354,6 +354,52 @@ def test_head_tail(self):
             self._compare(o.head(-3), o.head(7))
             self._compare(o.tail(-3), o.tail(7))
 
+    def test_sample(self):
+        o = self._construct(shape=10)
+        
+        # Run for all index types. 
+        for index in [ tm.makeFloatIndex, tm.makeIntIndex,
+                       tm.makeStringIndex, tm.makeUnicodeIndex,
+                       tm.makeDateIndex, tm.makePeriodIndex ]:
+            axis = o._get_axis_name(0)
+            setattr(o,axis,index(len(getattr(o,axis))))
+
+            # Check seeding -- run 10 times. 
+            for test in range(10):
```

Comment:
What are you trying to do here? We don't need (or want) to verify numpy's random number generator. Instead, a few sanity checks would be fine.